e2e/docker-compose.yml

```diff
@@ -26,6 +26,7 @@ services:
       --api.address=http://mysterium-api/v1
       --ether.client.rpc=http://geth:8545
       --keystore.lightweight
+      --ui.enable
```

Comment:
I'm not sure about this yet - do we really default to this set to enabled already? it's not really tested and currently not working.

Reply:
It's an e2e test, we are just asserting that UI server started and discovery works. 
But it is enabled for raspberry in a different place, I'll remove it for now.